server/data_provider_canvas_esb.rb

```diff
@@ -124,38 +124,48 @@ def canvasAPICalendarEventsURL(uniqname, canvas_courses)
     full_request_url
   end
 
+  # Create a string that expands the array of arguments and concatenates the array prefix to each one
+  def self.urlArrayParameters(array_prefix, parameters)
```

Comment:
This doesn't seem to be referenced anywhere except for its test, `test_urlArrayParameters`.